packages/p2p/src/components/order-details/order-details.scss

```
@@ -212,15 +218,14 @@ $card-width: 456px;
         line-height: 1.43;
         color: var(--text-general);
         padding: 1.6rem 2.4rem;
-        height: 7.2rem;
 
         @include mobile {
             width: 100%;
-            padding: 1.6rem;
+            padding: 0rem;
```

Although this is correct, a better way to handle this is to unset the previous given values like:

```suggestion
            padding: unset;
```